lib/workers/global/index.ts

```diff
@@ -73,6 +73,11 @@ export async function start(): Promise<0 | 1> {
     } else {
       logger.fatal({ err }, `Fatal error: ${String(err.message)}`);
     }
+    if (!config) {
+      // return early if we can't parse config options
+      logger.debug(`Missing config`);
```

Comment:
Is there already a fatal log before this?

Reply:
yes, it will log a fatal on line 74 above